### ISSUE TITLE
Add Drupal v6.x support

### ISSUE THREAD

**Issue by syrius01** [dreadlocked/Drupalgeddon2#8]
Hi !

Thanks for sharing those PoCs with the community :) I was wondering if you know how to exploit Drupal 6.* ?

So far; 
Drupal 7 is with /user/password
Drupal 8 is with /user/register

Thanks!

**Comment by dbjpanda** [comment]
Yes It will be really great if @dreadlocked can have a look on the D6 vulnerabilities. As there will be no official of D6. So we really need to manually test and patch our site.

**Comment by dreadlocked** [comment]
I'll try, can't promise!

**Comment by valicB** [comment]
Hi!. Any news about exploiting D6?

**Comment by syrius01** [comment]
From what I've found it seems that Drupal 6.X would be only for Cross Site scripting (this could be wrong). Just wanted to share.

**Comment by c3c** [comment]
Has anyone looked into whether Dripal6 would be actually vulnerable to this exploit already?

**Comment by stewpeed** [comment]
Someone need a little help regarding this : 
https://stackoverflow.com/questions/51203052/drupal-6-form-value-retrieve

**Comment by dreadlocked** [comment]
Closing this issue as Drupal 6 exploit is not inside the scope.
